### PR TITLE
feat: Added api contracts for orphaned tasks and departed users.

### PR DESCRIPTION
**Pull request by VinuB-Dev**
Date: 27-11-2024
Developer Name: Vignesh B S

---

## Issue Ticket Number
Closes Real-Dev-Squad/website-dashboard#888

## Description

Addition of api contracts for:
- `/users?departed=true&dev=true` - https://github.com/Real-Dev-Squad/website-backend/pull/2268
- `/tasks?orphaned=true&dev=true` - https://github.com/Real-Dev-Squad/website-backend/pull/2267

### Documentation Updated?

- [x] Yes
- [ ] No

### Under Feature Flag

- [x] Yes
- [ ] No

### Database Changes

- [ ] Yes
- [x] No

### Breaking Changes

- [ ] Yes
- [x] No

### Development Tested?

- [x] Yes
- [ ] No

## Screenshots
<details>
<summary>Screenshot 1</summary>

</details>

## Test Coverage
<details>
<summary>Screenshot 1</summary>

</details>

## Additional Notes

[Design Doc](https://docs.google.com/document/d/1ZYhy21DLHtTnXfKRlDH-7EVlKrMV2PyZs1_6czTozDk/edit?addon_store&tab=t.0#heading=h.cggvkhhvicot)

### CODE REVIEW

File: tasks/README.md [hunk 1]
```diff
@@ -82,6 +82,7 @@ Returns all the tasks
   - Optional: `size=[integer]` (`size` is the number of tasks requested per page. Range of value is 1-100. Default value is 5)
   - Optional: `next=[string]` (`next` is id of the document to get next page of results from that document)
   - Optional: `prev=[string]` (`prev` is id of the document to get prev page of results from that document)
+  - Optional: `orphaned=[boolean]` ( if departed is set to true with dev feature flag as true, it will return all the pending tasks which have assigned to users who have departed the discord server. )
```
Comment (AnujChhikara) [inline tasks/README.md:85]: `if departed` should be replaced by `if orphaned`

Comment (VinuB-Dev) [reply tasks/README.md]: Missed this, thanks for pointing out.
Have fixed it.

**Comment by pankajjs** [comment]
Why we have a lot of changes?

**Comment by VinuB-Dev** [comment]
> Why we have a lot of changes?

Prettier changes

**Review by AnujChhikara — APPROVED** [review]
LGTM

**Review by iamitprakash — APPROVED** [review]

**Pull request merged**